### ISSUE TITLE
logo refresh

### ISSUE THREAD

**Issue by skeep** [skeep/jscweb-2016#3]
Improve current logo.
Current logo 
![logo](https://cloud.githubusercontent.com/assets/425732/11649134/a9ac007e-9da1-11e5-88f4-2c221e13f738.png)
better on dark background - http://2015.jschannel.com/

**Comment by kailashverma** [comment]
![logo_](https://cloud.githubusercontent.com/assets/16349160/11894864/1954c66c-a5a0-11e5-94cf-3f026b0d8e34.png)
![logo_1](https://cloud.githubusercontent.com/assets/16349160/11894866/199be862-a5a0-11e5-9ac8-c93577f7991d.png)
![logo_3](https://cloud.githubusercontent.com/assets/16349160/11894867/19a1151c-a5a0-11e5-8897-1f9cae09eb01.png)
![logo_4](https://cloud.githubusercontent.com/assets/16349160/11894865/1996aa8c-a5a0-11e5-99a5-48d725a41312.png)

Here are JS new Logos

**Comment by skeep** [comment]
@kailashverma first 3 looks very close to JSConf original logo, would wana stay away from such similarity. Please refer to http://jsconf.com/

For the last one I am not a great fan for couple of reason, I think it's way complicated/complex than it should be. Also didn't like the angle. 
BTW I really liked the way you are thinking of changing the color profile.

**Comment by skeep** [comment]
I like 2 & 3 but I guess similarity will be the only reason I would prefer not to go ahead.

**Comment by kailashverma** [comment]
ok

**Comment by kailashverma** [comment]
![logo_5](https://cloud.githubusercontent.com/assets/16349160/11895456/d6a70f00-a5a4-11e5-8e04-c6b21f4fa9ca.png)
![logo_6](https://cloud.githubusercontent.com/assets/16349160/11895457/d6a93884-a5a4-11e5-9b62-7f8f924e709f.png)
![logo_7](https://cloud.githubusercontent.com/assets/16349160/11895455/d6a55e08-a5a4-11e5-9af6-f9d1a0bf59c9.png)
![logo_on-web](https://cloud.githubusercontent.com/assets/16349160/11895454/d6a302a2-a5a4-11e5-8884-8ed7015c23fe.png)

**Comment by kailashverma** [comment]
Have a look at these

**Comment by kailashverma** [comment]
Also look at LOGO on web , I ve put all different varients on the web

**Comment by kailashverma** [comment]
![logo_8](https://cloud.githubusercontent.com/assets/16349160/11896151/198c1040-a5aa-11e5-998e-d3d8d06840e2.png)

**Comment by kailashverma** [comment]
![logo_9](https://cloud.githubusercontent.com/assets/16349160/11897182/5f785b5a-a5b3-11e5-8dae-9c7e5ca8d312.png)

**Comment by skeep** [comment]
kinda like the last one ... although I can imaging some comments on it being too colorful! 

BTW what the `>` signify?

Just wondering how about trying another version with below changes 
- make the lines [vertical](http://www.imagingassociates.com.au/color/images/test-patterns/EBU_ColorBars-768x576.png)
- make the circle as rounded corner square.

**Comment by sohamkamani** [comment]
Just a suggestion ..  We can have a logo which matches js channel ... For example the picture on last years t shirt was literally a js "channel" , like a pipe turning code into apps , I thought that was really cool

**Comment by kailashverma** [comment]
> is a console sound 

Will be working on more versions tonight

**Comment by kailashverma** [comment]
>    is a console sign in JAVA programing

**Comment by kailashverma** [comment]
`>`  means console sign

**Comment by toamitkumar** [comment]
I agree with @sohamkamani we have to make sure we do not loose `JSChannel` from the logo.. especially the `TV` icon before the name is core to being a channel for everything JS

**Comment by pallavichauhan** [comment]
Just a suggestion
Lets all work on a brief for @kailashverma...he can then pick the brief and start working on different options. 
Please add your suggestions to the list of important elements/attributes that, according to you, should be interpreted from the logo. 

Logo Elements/ Attributes:
- Technology
- Channel

Other Info:
- emphasis on the word channel
- two variations - light and dark ( considering website and t-shirt design)

**Comment by kailashverma** [comment]
I have a question few Questions here
- Why is it called a CHANNEL ?? What exactly Channel means in this context ??
- If theres a good reason for channel , than How exactly Channel word or TV icon represent PROGRAMMING ( JAVA SCRIPT )  ?? , May be it should look more like a COMPUTER than a TV
- For me When I look at the logo its looks pretty RETRO or OLD School to me ,Is that the kind of 
  feeling we are trying to LOOK at ??

**Comment by skeep** [comment]
@kailashverma 
1. When we started we wanted to a tutorial video producer, hence JS**Channel**.
2. I think since it was meant to be view able on a screen we went with TV icon. I am not sure if we thought to connect it programming that time.
3. I agree, that's also another reason I started this initiative.

**Comment by pallavichauhan** [comment]
@skeep Please find some new logos created by Sachin. also, please add him to this github.

![logo-2](https://cloud.githubusercontent.com/assets/16237745/12610780/820d08fc-c50f-11e5-892a-9350cd282a85.jpg)

**Comment by pallavichauhan** [comment]
![logo-1](https://cloud.githubusercontent.com/assets/16237745/12610791/8e4b821a-c50f-11e5-90ad-d1bc023715e9.jpg)

**Comment by pallavichauhan** [comment]
![logo-3](https://cloud.githubusercontent.com/assets/16237745/12610805/96e1dbfe-c50f-11e5-98bc-7371c0bee174.jpg)

**Comment by pallavichauhan** [comment]
![logo-4](https://cloud.githubusercontent.com/assets/16237745/12610811/9dc29972-c50f-11e5-8e0d-48263ca9d9d0.jpg)
